.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -31,6 +31,7 @@
     <inspection_tool class="Convert2Lambda" enabled="true" level="WEAK WARNING" enabled_by_default="true" />
     <inspection_tool class="Convert2MethodRef" enabled="true" level="WEAK WARNING" enabled_by_default="true" />
     <inspection_tool class="CssUnusedSymbol" enabled="true" level="WEAK WARNING" enabled_by_default="true" />
+    <inspection_tool class="Duplicates" enabled="false" level="WEAK WARNING" enabled_by_default="false" />
```

Comment:
Please, remove this change from the first commit. It does not relate to other changes, and I think that changing IDEA inspection is a matter of a separate discussion.

Reply:
Alexey, re the IDEA inspection my error, I will fix that.  I will re-enable checkstyles and fix that too.. sorry.  I got the code from openjdk8.  Is that acceptable?  Perhaps I need to include their license somewhere?

Reply:
You can't include any code from OpenJDK, since it's licensed under GPL, which is not compatible with Apache license.

Reply:
Ok that makes it a bit tricky.  I will see if Harmony or GWT has these and let you know.